core/themes/bartik/css/style.css

```diff
@@ -569,9 +569,19 @@ ul.tips {
   background: rgba(255, 255, 255, 0.7);
   height: 2.4em;
   line-height: 2.4em;
-  padding: 0 0.8em;
```

Comment:
@klonos Add this line back in and it seems to fix things in all situations (from my testing).

Reply:
Thanks @BWPanda but I wasn't able to reproduce this. The fork from which this PR was filed has been nuked, so I'm replacing it with https://github.com/backdrop/backdrop/pull/3487